release_guidelines.md

```diff
@@ -8,73 +8,96 @@ We follow [semver](http://semver.org/) for versioning. Given a version number MA
 
 ## Creating a Release Candidate
 
-An example `<VERSION_RC>` could be `2.0.0-rc.1`
+An example release candidate version `<VERSION_RC>` could be `2.0.0-rc.1`
 
-* Create a release branch: `release/<version>`. Use the final version rather than a release candidate in the branch name.
-* Update the SDK package version in `package.json` to `<VERSION_RC>`
-* Update the change log following [this](http://keepachangelog.com/)
-* Increment `BASE_32_VERSION` in `webpack.config.babel.js` e.g. `AA` => `AB`
-  * You only need to do this once per release, *not* per release candidate
-* Build the dist files with `npm run build`
-* Run `aws s3 sync ./dist s3://onfido-assets-production/web-sdk/<BASE_32_VERSION>/ --exclude "*.html" --exclude "*.map" --acl public-read --delete`
-* [Deploying the release to S3 production](#deploying-the-release-to-S3-production)
-  * Run `aws s3 sync ./dist s3://onfido-assets-production/web-sdk-releases/<VERSION_RC> --exclude "*.html" --exclude "*.map" --acl public-read --delete`
-* Commit the above using the version as the commit message
-* Create a pull request from the release branch into master
-* [Update JSFiddle demo](#update-jsfiddle-demo)
-* On the release branch run `npm publish --tag next`
-  * Get the credentials from One Login
-  * Read about npm and release candidates [here](https://medium.com/@mbostock/prereleases-and-npm-e778fc5e2420)
-* Check the `latest` tag had not moved with `npm dist-tag ls onfido-sdk-ui`
-* Check you can install the package with `npm install onfido-sdk-ui@<VERSION_RC>`
-* Create a git tag with rc version on github, i.e `2.0.0-rc.1`:
-  * Pointing to the release commit on the release branch
-  * The release tag and title should be the version
-  * The description should be the change log entry for the release
-* Test the SDK deployment on surge associated with the PR
+1. Make sure all features for the forthcoming release have been merged to `development` branch.
+2. Checkout and pull `develop` branch
+    - `git checkout develop && git pull`
+3. Make sure README.md file has been updated
+4. Make sure CHANGELOG.md has been updated
+    - with new version number
+    - with all Public, Internal and UI changes
+5. Make sure MIGRATION.md has been updated, if applicable.
+6. Update the SDK package version in `package.json` to `<VERSION_RC>`
+7. Increment `BASE_32_VERSION` in `webpack.config.babel.js` e.g. `AA` => `AB`
+    - do it only ONCE per release, *not* per release candidate
```

Comment:
Maybe this should read:
`do it only if a breaking change is introduced between SDK and cross device client. This must be done only ONCE per release, *not* per release candidate`